### PR TITLE
SC-5644 - a11y topbar

### PR DESCRIPTION
**Pull request by haniapalac**
# Description

## Links to Tickets or other pull requests

https://ticketsystem.schul-cloud.org/browse/SC-5644

## Changes

Changes mainly for topbar, visual focus added

## Screenshots of UI changes

image

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)
https://docs.schul-cloud.org/display/CAP/Acceptance+Criteria

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -1600,11 +1609,13 @@
...
-				"qrCode" : "QR-Code",
-				"printQrCode" : "QR-Code drucken",
-				"notifications" : "Benachrichtigungen"
+				"currentUser": "aktueller Benutzer:",
+				"fullscreen": "Vollbildmodus",
+				"thereIsAProblem": "Es gibt ein Problem",
+				"qrCode": "QR-Code",
+				"printQrCode": "QR-Code drucken",
+				"notifications": "Benachrichtigungen",
+				"search": "search"
```
Comment (luizzaly) [inline locales/de.json]: please change to "Suche"

File: locales/de.json [hunk 2]
```diff
@@ -1140,6 +1143,11 @@
 				"teams": "Teams"
 			}
 		},
+		"skipLinks": {
+			"mainContentSkipLink": "Hauptbereich",
+			"topNavbarSkipLink": "Navigation",
+			"footerSkipLink": "Footerc"
```
Comment (luizzaly) [inline locales/de.json]: I guess it's a typo here > Footerc

Comment (haniapalac) [reply locales/de.json]: Fixed :-)

File: locales/de.json (hunk 1 above)
Comment (haniapalac) [reply locales/de.json]: done :-)

File: locales/de.json [hunk 3]
```diff
@@ -1617,12 +1626,15 @@
...
-				"notifications": "Benachrichtigungen",
+
+			"label" : {
+				"currentUser": "aktueller Benutzer:",
+				"fullscreen": "Vollbildmodus",
+        		"notifications": "Benachrichtigungen",
 				"printQrCode": "QR-Code drucken",
 				"qrCode": "QR-Code",
-				"thereIsAProblem": "Es gibt ein Problem"
+				"search": "search",
```
Comment (luizzaly) [inline locales/de.json:1636]: please change to Suche

File: locales/de.json [hunk 4]
```diff
@@ -1113,6 +1113,8 @@
 		"link": {
 			"openInNewTab": "in neuem Tab öffnen"
 		},
+		"mobileNavMenu": "mobile nav toggle",
+		"openInNewTabWarning": "Opens in new tab",
```
Comment (haniapalac) [inline locales/de.json:1117]: @luizzaly can you please provide a proper German translation here? Thanks.

Comment (luizzaly) [reply locales/de.json:1117]: mobileNavMenu: "Mobiles Navigations Menu"
openInNewTabWarning:"Wird in einem neuem Tab geöffnet"

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5892666&bid=19295397)

File: views/lib/help_menu.hbs [hunk 5]
```diff
@@ -1,61 +1,65 @@
...
-                    </span>
-                </li>
+	<div class="dropdown minimal-button">
+			<a class="btn btn-secondary btn-thin dropdown-toggle"
+		   	   href="" title="{{$t "lib.help_menu.label.help"}}" aria-label="{{$t "lib.help_menu.label.help"}}"
+		       data-toggle="dropdown">
+				<i class="fa fa-question" aria-hidden="true"></i>
+			</a>
+		<div class="dropdown-menu">
+			<ul class="content list-group">
```
Comment (luizzaly) [inline views/lib/help_menu.hbs:9]: atm I am not able to navigate through the dropdown via arrow keys, just wondering if this is needed..

Comment (haniapalac) [reply views/lib/help_menu.hbs:9]: As decided this should be fixed in a separate PR -> https://ticketsystem.schul-cloud.org/browse/SC-5858 
I will mark this comment as resolved then.

**Pull request merged**